### PR TITLE
feat(react-ui-kit): Implement new tooltip

### PR DESCRIPTION
**Pull request by thisisamir98**
## Pull Request Checklist

- [X] My code is covered by tests
- [X] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

## Demo

image

### CODE REVIEW

**Comment by thisisamir98** [comment]
Done in https://github.com/wireapp/wire-web-packages/pull/5012/commits/79fcdc8f959bfb9e617fce10a2637e5e7149dc7a

image

File: packages/react-ui-kit/src/Form/Tooltip.tsx [hunk 1]
```diff
@@ -21,64 +21,91 @@ import * as React from 'react';
...
-    transform: left || right ? `translateX(${left ? -16 : 16}px)` : `translateY(${bottom ? -16 : 16}px)`,
-    transition: 'all 0.15s ease-in-out',
+    boxSizing: 'border-box',
+    display: 'block',
+    margin: '0 auto',
+    padding: '4px 8px',
+    backgroundColor: theme.Tooltip.backgroundColor,
+    color: theme.Tooltip.color,
+    borderRadius: 4,
+    fontSize: '0.8em',
```
Comment (V-Gira) [inline packages/react-ui-kit/src/Form/Tooltip.tsx:50]: can't this be something like `theme.fontSizes.small`  (0.75rem)? 
rem would let it scale with the font size slider, design was talking about using tooltips for accessibility, that would help

**Comment by thisisamir98** [comment]
done in https://github.com/wireapp/wire-web-packages/pull/5012/commits/40e3229ce35ce08816b0ecbf377163f6e2c0ec4f

image

image

File: packages/react-ui-kit/src/Form/Tooltip.tsx (hunk 1 above)
Comment (thisisamir98) [reply packages/react-ui-kit/src/Form/Tooltip.tsx]: sure, done in:
https://github.com/wireapp/wire-web-packages/pull/5012/commits/778bca21cb9eaea35dca9f16be11ff061b68fdad

**Review by V-Gira — APPROVED** [review]

**Pull request merged**